### PR TITLE
Update high-performance-tidb-challenge.md

### PR DESCRIPTION
**Pull request by WT-Liu**
update participation notes

### CODE REVIEW

**Comment by WT-Liu** [comment]
@dcalvin PTAL~

File: challenge-programs/high-performance-tidb-challenge.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Glad you are interested! Before you start hacking, these are something you shoul
 ### About team-up
 
 * You need to team up before submitting the [application form](https://forms.pingcap.com/f/high-performance-challenge-en). One team could only select one workload for optimization.
-* Up to five participants are allowed in a team; cross team-up is not allowed.
+* Up to six participants are allowed in a team (The mentor and the team leader are included). Cross team-up is not allowed.
```
Comment (dcalvin) [inline challenge-programs/high-performance-tidb-challenge.md:27]:
```suggestion
* Up to six participants are allowed in a team (mentor included). Cross team-up is not allowed.
```

File: challenge-programs/high-performance-tidb-challenge.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ Glad you are interested! Before you start hacking, these are something you shoul
 ### About mentors
 
 * Only one mentor is allowed in a team and one mentor could only provide instructions to one team.
+* The mentor is recognized as a participant in the team. In addition to the mentor prize, the mentor is eligible to claim team prize.
```
Comment (dcalvin) [inline challenge-programs/high-performance-tidb-challenge.md:40]:
```suggestion
* Besides the regular mentoring instructions, the mentor is also allowed to claim the tasks submit PRs. As such,  in addition to the mentor prize, the mentor is eligible to share the team prize.
```

**Comment by WT-Liu** [comment]
@dcalvin PTAL again~ Thanks!

**Review by dcalvin — APPROVED** [review]

**Comment by dcalvin** [comment]
@gingerkidney @AndreMouche PTAL

**Comment by gingerkidney** [comment]
LGTM

**Review by winkyao — APPROVED** [review]

**Pull request merged**